deps.sh

```diff
@@ -0,0 +1,2 @@
+tarantoolctl rocks install luatest 1.0.1
```

Comment:
why not `tt`?

Reply:
There were issues with the installation, it seems like setup-tarantool doesn't install it.

Reply:
i tried to make CI works, i will clean up . i will try to bump setup-tarantool and add tt